static/app/components/codeSnippet.tsx

```diff
@@ -146,22 +176,49 @@ const Header = styled('div')<{hasFileName: boolean}>`
       width: max-content;
       height: max-content;
       max-height: 100%;
-      padding: ${space(1)};
+      padding: ${space(0.5)};
     `}
 `;
 
 const FileName = styled('p')`
   ${p => p.theme.overflowEllipsis}
+  padding: ${space(0.5)} ${space(0.5)};
   margin: 0;
+  width: auto;
+`;
+
+const TabsWrapper = styled('div')`
+  padding: 0;
+  display: flex;
+`;
+
+const Tab = styled('button')<{isSelected: boolean}>`
+  box-sizing: border-box;
+  display: block;
+  margin: 0;
+  border: none;
+  background: none;
+  padding: ${space(1)} ${space(1)};
+  color: #80708f;
+  ${p =>
+    p.isSelected
+      ? `border-bottom: 3px solid ${p.theme.purple300};
+      padding-bottom: 5px;
+      color: #E0DCE5;`
```

Comment:
Hardcoded the colors here as I was not sure how to set them otherwise if the theme should not affect them 🤔